src/commands/scanner/run.ts

```diff
@@ -195,17 +196,17 @@ export default class Run extends ScannerCommand {
 		if (this.flags.format && this.flags.outfile) {
 			const derivedFormat = this.deriveFormatFromOutfile();
 			// For validation purposes, treat junit as xml.
-			const chosenFormat = this.flags.format == 'junit' ? 'xml' : this.flags.format;
+			const chosenFormat = this.flags.format as string == 'junit' ? 'xml' : this.flags.format as string;
```

Comment:
Yes, because the problem isn't that `chosenFormat` has no clear type, but rather that `this.flags.format` has no clear type. It does turn out that only the latter `as string` is necessary, though.